pointers/pointers.c

```diff
@@ -33,7 +47,14 @@ char *find_char(char *str, int c)
     Do not just use the `strcpy` function from the standard library.
 */
 void string_copy(char *x, char *y)
-{
+{   
+  int c = 0;
+
+    while (y[c] != '\0') {
+    x[c] = y[c];
+    c++;
+  }
+    x[c] = '\0';
 
```

Comment:
It looks like you're using array indices to copy the string, which is not the standard way of doing this but it tests out. I think you are avoiding the use of the *, which I can't blame you for because it can mean different things depending on its location (which is why pointers are so confusing) but it is worth trying.